src/compiler/factory.ts

```diff
@@ -101,7 +101,7 @@ namespace ts {
         return node;
     }
 
-    function createLiteralFromNode(sourceNode: StringLiteral | NumericLiteral | Identifier): StringLiteral {
+    function createLiteralFromNode(sourceNode: StringLiteralLike | NumericLiteral | Identifier): StringLiteral {

```

Comment:
This isn't internal, but it uses `StringLiteralLike`. You can technically make an internal overload for it, or use `NoSubstitutionTemplateLiteral`

Reply:
This isn't exported so it won't show up in the API. API baseline tests would have told us if we tried to use an internal type in a public api.